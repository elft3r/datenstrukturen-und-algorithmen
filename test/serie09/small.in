4
4 5 1 4 3 2 3 4 3 1 9 4 2 1 1 2 5
10 15 1 2 1 2 3 2 1 4 7 2 5 2 3 6 3 3 7 5 4 5 4 5 6 3 6 7 2 4 8 1 5 9 3 6 9 4 7 10 6 8 9 5 9 10 3
9 14 1 2 10 1 3 11 2 4 11 2 5 11 5 6 13 3 4 10 4 6 12 4 7 5 3 7 4 3 8 5 8 7 5 8 9 4 9 7 3 6 7 11
14 14 14 2 4 14 3 3 14 8 9 1 3 3 3 7 8 11 7 3 1 4 3 12 3 9 4 6 9 8 7 7 13 5 4 5 8 8 9 4 6 6 10 1
