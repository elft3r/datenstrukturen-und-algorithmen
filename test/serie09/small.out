8
21
59
68
